main.go

```diff
@@ -582,9 +615,21 @@ func (p *gateway) scheduleStatCollection(ctx context.Context, scheduler *sfxclie
 	return finishedContext, cancelFunc
 }
 
-func (p *gateway) setupForwardersAndListeners(ctx context.Context, loader *config.Loader, loadedConfig *config.GatewayConfig, logger log.Logger, scheduler *sfxclient.Scheduler) (signalfx.Sink, map[string]http.Handler, error) {
+func (p *gateway) scheduleStatCollectionWithDefaultDelay(ctx context.Context, scheduler *sfxclient.Scheduler, loadedConfig *config.GatewayConfig, multiplexer signalfx.Sink) (context.Context, context.CancelFunc) {
+	scheduler.ReportingDelayNs = (time.Second * 10).Nanoseconds()
+	finishedContext, cancelFunc := context.WithCancel(ctx)
+	if loadedConfig.InternalMetricsReportingDelayDuration != nil && *loadedConfig.InternalMetricsReportingDelayDuration != 0 {
+		scheduler.Sink = multiplexer
+		scheduler.ReportingDelayNs = loadedConfig.InternalMetricsReportingDelayDuration.Nanoseconds()
+	} else {
+		p.logger.Log("skipping internal metrics stat keeping")
```

Comment:
again, this isn't a thing.  this is not a skippable thing. If it's not configured it's 10 seconds.